apps/web-giddh/src/app/userDetails/components/company-details-sidebar/company-details-sidebar.component.scss

```diff
@@ -0,0 +1,127 @@
+@import "../../../../assets/css/color.scss";
+.company-list {
+    color: $dark-gray;
+    list-style-type: none;
+}
+:host {
+    position: fixed;
+    left: auto;
+    top: 0;
+    right: 0;
+    bottom: 0;
+    max-width: 760px;
+    width: 100%;
+    z-index: 99999;
+}
+
+:host.in #close {
+    display: block;
+    position: absolute;
+    right: 10px;
+    top: 16px;
+    z-index: 5;
+    border: 0;
+    border-radius: 0;
+    background: 0 0;
+    color: $white;
+}
+
+:host .container-fluid {
+    padding-left: 0;
+    padding-right: 0;
+}
+
+:host .aside-pane {
+    max-width: 760px;
+    width: 100%;
+}
+
+.custom-tags {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    flex-wrap: wrap;
+
+    span {
+        border: 1px solid #ddd;
+        padding: 5px 8px;
+        margin: 10px 10px 12px 0px;
+        background-color: #ddd;
+    }
+}
+.input-group {
+    width: 100%;
+}
+.aside-pane-form {
+    background-color: #fff;
+    box-shadow: 0px 3px 6px #0000001a;
```

Comment:
Use color variables

Reply:
in color, border, & background-color property we  using var color name 
we are not using var color name in box-shadow